pkg/p2p/wire/message/reduction.go

```diff
@@ -0,0 +1,208 @@
+package message
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/dusk-network/dusk-blockchain/pkg/core/consensus/header"
+	"github.com/dusk-network/dusk-blockchain/pkg/p2p/wire/encoding"
+	"github.com/dusk-network/dusk-blockchain/pkg/util"
+	"github.com/dusk-network/dusk-crypto/bls"
+	"github.com/dusk-network/dusk-wallet/v2/key"
+)
+
+type (
+	// the consensus.
```

Comment:
This comment looks a bit unfinished. :)

Reply:
I am not sure what you have against postmodern expressionism in comments. Anyway, I will cha.